lib/plugin-registry.coffee

```diff
@@ -35,22 +36,25 @@ module.exports =
     @plugins[code] = plugin
 
     if @enabled
-      @addConfigForPlugin code, plugin, key
+      @addConfigForPlugin(code, plugin, key)
       @observePlugin code, plugin, key
 
   addConfigForPlugin: (code, plugin, key) ->
+    return if atom.config.get(key) != undefined
```

Comment:
I think it should register the config for the plugin even if is already set a value for it.